hub/config.go

```diff
@@ -32,6 +32,9 @@ func SetConfigDefaults(v *viper.Viper) {
 	v.SetDefault("metrics", false)
 	v.SetDefault("metrics_login", "mercure")
 	v.SetDefault("metrics_password", "")
+
+	v.SetDefault("telemetry.enabled", false)
```

Comment:
We don't use dot in keys yet (but a underscore). Also can we keep "metrics"? telemetry refers to another thing to me (the ability for the editor to gather stats, and we currently don't do this).

Reply:
I chose telemetry, because then it's extensible in the future with things like distributed tracing. Also, I find dot notated config (nested yaml). I can change it, if it doesn't make sense or consistency has a priority now.

Reply:
Yes please change it. Consistency as priority now and I fear the dot will not work with other supported config formats.